ISSUES.md

```diff
@@ -0,0 +1,54 @@
+**If you are reporting *any* crash or *any* potential security issue, *do not*
```

Comment:
Why do we have both ISSUES_TEMPLATE.md and ISSUES.md after this PR @mattklein123?

Reply:
I see, this is essentially a rough version of what GH now supports through explicit template choosers.

Reply:
Right, this is pre new stuff. We can definitely make this all better.